### PR TITLE
poc: run integration tests within a podman pod

### PR DESCRIPTION
**Pull request by mhoffm-aiven**
This PR is a POC to run integration tests within a podman pod

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -97,23 +96,13 @@ def pytest_addoption(parser, pluginmanager) -> None:  # pylint: disable=unused-a
...
-    if not re.match(VERSION_REGEX, kafka_version):
-        msg = "Provided Kafka version has invalid format {kafka_version} should match {VERSION_REGEX}"
-        raise ValueError(msg)
-
-    if kafka_bootstrap_servers is not None and log_dir is not None:
-        msg = f"{KAFKA_BOOTSTRAP_SERVERS_OPT} and {LOG_DIR_OPT} are incompatible options, only provide one of the two"
-        raise ValueError(msg)
-
-    if kafka_bootstrap_servers is not None and kafka_version is not None:
+    if kafka_bootstrap_servers is None:
```
Comment (hackaugusto) [inline tests/conftest.py:105]: There seems to be still an error:

```
E   ValueError: --kafka-bootstrap-servers and --kafka-version are incompatible options, only provide one of the two
```

File: tests/conftest.py [hunk 2]
```diff
@@ -97,23 +96,13 @@ def pytest_addoption(parser, pluginmanager) -> None:  # pylint: disable=unused-a
...
     kafka_bootstrap_servers = request.config.getoption("kafka_bootstrap_servers")
-    log_dir = request.config.getoption("log_dir")
-    kafka_version = request.config.getoption("kafka_version")
     registry_url = request.config.getoption("registry_url")
     rest_url = request.config.getoption("rest_url")
     server_ca = request.config.getoption("server_ca")
 
     has_external_registry_or_rest = registry_url or rest_url
 
-    if not re.match(VERSION_REGEX, kafka_version):
```
Comment (hackaugusto) [inline tests/conftest.py:108]: Now this is causing a lint error:

```
tests/conftest.py:6:1: F401 're' imported but unused
```

Please run `pre-commit` there are also some issues with the `.sh` file

File: scripts/integration.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```
Comment (hackaugusto) [inline scripts/integration.sh:1]: To properly test this, it should be enabled on the github actions, here:

https://github.com/aiven/karapace/blob/eb5a7720475b66de5011722db4799e3815fdacaa/.github/workflows/tests.yml#L41-L43

**Review by tvainika — CHANGES_REQUESTED** [review]
Didn't do deeper review, maybe this PR should have been opened in draft mode to not request reviews yet?

**Comment by mhoffm-aiven** [comment]
> Didn't do deeper review, maybe this PR should have been opened in draft mode to not request reviews yet?

absolutely, sorry for the confusion

**Comment by jjaakola-aiven** [comment]
I queried the status from @mhoffm-aiven and I am closing this as not relevant.

**Pull request closed without merging**